src/features/JoinView/GuestQR.js

```diff
@@ -57,13 +66,27 @@ export default function GuestQR() {
       .catch((error) => {
         console.log('Error when adding GUEST into the database');
       });
+    let hostName = sampleData[1];
+    let diningPlaceName = sampleData[2];
+    let diningPlaceId = sampleData[3];
 
+    alert(`Session Id: ${sessionId} \n  HostName: ${hostName} \n
+    Restaurant Name: ${diningPlaceName} \n RestaurantID: ${diningPlaceId}` );
+    dispatch({ type: 'SET_RESTAURANT_ID', restaurantId: diningPlaceId });
+    dispatch({ type: 'SET_RESTAURANT_NAME', restaurantName: diningPlaceName });
+    //document ID- from query****
     //***********@TODO----Once we get the  information----************
     // HOST needs to be updated with guest name - in real time (firestore)
     // HOST will get notification (current session -> summary )that someone wants to join the session?
     // After HOST 'allow' the guest entry, update in real time (firestore snapshot), update conetxt api under guest[{name:Greg}]
     // Guest get confirmation update ('waiting' -> 'allowed'), redirect to the (current -> menu)
+    if (scanned) {
+      return;
+    }
+  };
 
+  const reRoute = async()=>{
+    await delay(500);
```

Comment:
What does this piece of code do?

Reply:
hahaha apparently nothing.. I will double check with Crystal

Reply:
Nothing related with the google login - 
I am working on the QR code, and then make a branch to work on google login